x-pack/plugins/cases/public/containers/api.test.tsx

```diff
@@ -855,10 +856,8 @@ describe('Cases API', () => {
     const caseConnectors = getCaseConnectorsMockResponse();
     const connectorCamelCase = caseConnectors['servicenow-1'];
 
-    const snakeCaseConnector = {
-      ...connectorCamelCase,
-      push: { ...connectorCamelCase.push, externalService: basicPushSnake },
-    };
+    const snakeCaseConnector = cloneDeep(connectorCamelCase);
```

Comment:
When Kibana is upgraded to TS 4.7 we can use [`structuredClone`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) 🙂. Unfortunately, we cannot use it now.

Reply:
Oh cool, seems like from my local browser testing that `cloneDeep` might be faster 🤷‍♂️ tradeoff between that and not having to import lodash I suppose.